pkgs/by-name/jh/jhentai/pubspec.lock.json

```diff
@@ -4,28 +4,34 @@
       "dependency": "transitive",
       "description": {
         "name": "_fe_analyzer_shared",
-        "sha256": "0b2f2bd91ba804e53a61d757b986f89f1f9eaed5b11e4b2f5a2468d86d6c9fc7",
-        "url": "https://pub.dev"
+        "sha256": "16e298750b6d0af7ce8a3ba7c18c69c3785d11b15ec83f6dcd0ad2a0009b3cab",
+        "url": "https://pub.flutter-io.cn"
```

Comment:
Do we want to use the chinese flutter proxy?

Reply:
The performance for people outside of china will probably be much worse than for pub.dev.